packages/v4/src/content/design-guidelines/components/truncate/truncate.md

```diff
@@ -0,0 +1,47 @@
+---
+id: Truncate design guidelines
+section: components
+---
+
+A **truncate** is a tool used to shorten numeric and non-numeric character strings, typically when the string overflows its container. 
+
+## Elements
+* Content: text string to truncate
+* Position: where the text will be truncated from, ie. front, middle, or end
+* Tooltip hover: displayes full-text string 
+* Ellipsis: indicates a truncate is used
+
+## Usage
+Truncated items are indicated by an ellipsis (...) and are used when trying to avoud multiple lines of text or when a container is being overflowed by content. Truncation should only be used when 3 or more characters are being represented and there are still at least 4 non-truncated characters displayed. Truncated items should always incldue a tooltip on hover, showcasing the full string sequence. 
+* **Breadcrumbs**: Use a breadcrumbs truncate when there isn’t enough room to display the entire breadcrumbs list, or as a way of managing relevance.
+<img src="./img/breadcrumb.png" alt="image showing breadcrumbs truncation" width="353" />
+* **Long URL's**:  Use a URL truncate when the url overflows its container
+<img src="./img/longurl.png" alt="image showing long url truncation" width="442" />
+* **Description text/alerts**: Use a truncate for description text to create more room on the page
+<img src="./img/truncatealert.png" alt="img showing alert truncate" width="632" />
+* **Table inputs**: Use truncation when the data within a table overflows its container
+ <img src="./img/tabletruncate.png" alt="image showing table truncate" width="374" />
```

Comment:
This image looks really small to me (?)  double check the width just to make sure, otherwise no worries!